httputil/respbuf.go

```diff
@@ -12,20 +12,26 @@ import (
 	"strconv"
 )
 
+// ResponseBuffer is the current response being composed by its owner.
+// It implements the http.ResponseWriter interfaces: Write, WriteHeader, and Header,
+// as well as io.WriterTo interface.
 type ResponseBuffer struct {
 	buf    bytes.Buffer
 	status int
 	header http.Header
 }
 
+// Write implements the ResponseWriter interface on a ResponseBuffer
```

Comment:
please remove "on a ResponseBuffer" and add a period to the end of these comments.